### DISCUSSION TITLE
What distinguishes this from existing projects?

### DISCUSSION THREAD

**Discussion by mysterybear** [stoatchat/discussions#130] [General]
~~As compared with Telegram, Signal, Riot etc?~~

~~If anyone else is like me, I suggest adding this to the main website, it was the first thing I was looking for (the "start with why" meme)~~

Sorry I'm dumb, this is like an alternative to Slack/Discord etc not Telegram/Signal etc, lol. I think this was my own problem sorry for that :D

Even so it might be a question worth addressing on the landing page!

**Comment by dreaddr** [comment]
What I want to know is if Revolt is a [matrix] client or not. Is it? If not, are there bridges like on [matrix]?

**Reply by erkinalp** [reply]
Revolt is more like Slack than Matrix.

**Reply by dreaddr** [reply]
Right... that doesn't help me much then
